README.md

```diff
@@ -26,6 +26,7 @@ An [Ansible AWX](https://github.com/ansible/awx) operator for Kubernetes built w
          * [Containers Resource Requirements](#containers-resource-requirements)
          * [LDAP Certificate Authority](#ldap-certificate-authority)
          * [Persisting Projects Directory](#persisting-projects-directory)
+         * [Custom Setting Configuration](#tower-custom-settings)
```

Comment:
```suggestion
         * [Custom Setting Configuration](#custom-settings)
```

Reply:
A little on the nitpicky side, but we should be careful about places we say "Tower" 

The variable names are fine (Not really - we need to go through and remove all references to `tower_` at some point soon, but I'd prefer to follow the existing pattern to make find/replace easier)